### ISSUE TITLE
 Develop a migration strategy and cut over for Secret/cert Automation Work 

### ISSUE THREAD

**Issue by meganhicks** [department-of-veterans-affairs/abd-vro#3356]
After establishing secret automation for all services in lower environments, we need to develop a comprehensive plan for the production cutover. This ticket will cover the migration strategy and the cutover process to production. 

**Acceptance criteria**

1. There is a plan for in place for how we plan to deploy secret automation to production environments. 
2. Secret Automation is in prod for all services. 
3. Prod test is not accounted for.

**Comment by brostk** [comment]
Noting something I found during the `svc-bie-kafka` migration:

There is a file called `set-env-secrets.src` which is executed on pod startup by `entrypoint-wrapper.sh`.  That script pulls in the legacy bundled secret and decodes it, then exports all of the environment variables which we've migrated over to Vault.  After an app has been migrated, the environment variables have already been set by our deployment template - this script being executed means those values we've injected will be overwritten by the contents of that legacy secret.  Part of our final cut over needs to involve removing the call to that script from `entrypoint-wrapper.sh` to ensure we're using the correct secrets we've configured in Vault, and removing the old secret from Kubernetes.
